File: 1064.py
n = []
for i in range (0,6):
    num=(float(input()))
    n.append(num)
pos = 0
soma = 0.0
for v in n:
    if(v>0):
        pos += 1
        soma += v
media = soma / float(pos)
print("%d valores positivos" %pos)
print("%.1f" %media)

File: 1064.c
#include <stdio.h>
 
int main() {
 
 int i = 1, y = 0;
 double x, media, soma = 0;
 
 
 while(i <= 6){
	 scanf("%lf", &x);
	 if (x > 0.0){
	 	y++;
  soma = soma + x;
	 }
	 i++;
 }
 media = soma/y;
 
 printf("%d valores positivos\n", y);
 printf("%.1lf\n", media);
 
    return 0;
}